cypress/support/helpers.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { PLUGIN_NAME } from './constants';
+import sample_field_mappings from '../fixtures/sample_field_mappings.json';
+import sample_index_settings from '../fixtures/sample_index_settings.json';
+
+export const createDetector = () => {
```

Comment:
This is used in findings_spec

Reply:
this is a cousin of creation portion of detectors_spec

Reply:
Does the `findings_spec` need to use this helper method, or could it make use of `cy.createDetector`, and edit the `sample_detector` JSON like I did in [this test](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/135/files#diff-c25ee6dc498b8de8bbb3e0105f01d4cd41d5c2a40b7f409c4c9c42faa3c8bfabR23)?

Reply:
findings needs some fixing, but I can add rules back to this branch if you like.  I figured since your original hasn't been merged it would have no effect on main, then if your PR is unblocked it will be added back at that point.  Is this incorrect?

Reply:
Yeah, I'd say add the `rules_spec` back in. That will allow those tests to run for this PR. We want our automated testing to be as robust as possible.

Reply:
of course.

Reply:
agreed

Reply:
the reason I created this helper is because createDetector would result in no finding.  I will sub it back in to see if anything has changed.